### ISSUE TITLE
TODO: Refactoring PrismaClientKnownRequestError

### ISSUE THREAD

**Issue by stijnvanderlaan** [demonsters/prisma-mock#19]
I'm encountering issues with importing `PrismaClientKnownRequestError` from Prisma, which are causing a problem #16. To address this, I'm considering implementing a custom version of `PrismaClientKnownRequestError`. While this approach would have the disadvantage of not being able to compare instances, it would resolve the issues with backwards compatibility. Since this is a backwards incompatible change, I plan to group it with other backward incompatible changes.

**Comment by leighman** [comment]
Can you make it possible for the user to provide a way of constructing it or something?
We use the mock to test code that matches on `PrismaClientKnownRequestError` with `instanceof`.
Which seems to be broken in 4.10 anyway?

**Comment by stijnvanderlaan** [comment]
I think that will still work in 4.10, don't know for sure. 

I guess I could add them as a dependancy, like so:
```js
import * as PrismaRuntime from "@prisma/client/runtime"
createPrismaMock({
  datamodel: Prisma.dmmf.datamodel,
  runtime: PrismaRuntime
})
```

Note: The parameters will have to be refactored (#20), because it would otherwise be a mess.

**Comment by stijnvanderlaan** [comment]
Fix in v0.6.0
